sdk/storage/azure-storage-file-share/migration_guide.md

```diff
@@ -0,0 +1,1208 @@
+            <td width="107" valign="top">
+                <p align="right">
+                    ShareDirectoryClient
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td width="242" valign="top">
+                <p align="right">
+                    list_directories_and_files
+                </p>
+            </td>
+            <td width="111" valign="top">
+                <p align="right">
+                    FileService
+                </p>
+            </td>
+            <td width="163" valign="top">
+                <p align="right">
+                    list_directories_and_files
+                </p>
+            </td>
+            <td width="107" valign="top">
+                <p align="right">
+                    ShareDirectoryClient or ShareClient
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td width="242" valign="top">
+                <p align="right">
+                    list_handles
+                </p>
+            </td>
+            <td width="111" valign="top">
+                <p align="right">
+                    FileService
+                </p>
+            </td>
+            <td width="163" valign="top">
+                <p align="right">
+                    list_handles
+                </p>
+            </td>
+            <td width="107" valign="top">
+                <p align="right">
+                    ShareDirectoryClient or ShareFileClient
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td width="242" valign="top">
+                <p align="right">
+                    close_handles
+                </p>
+            </td>
+            <td width="111" valign="top">
+                <p align="right">
+                    FileService
+                </p>
+            </td>
+            <td width="163" valign="top">
+                <p align="right">
+                    close_all_handles
+                </p>
+            </td>
+            <td width="107" valign="top">
+                <p align="right">
+                    ShareDirectoryClient or ShareFileClient
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td width="242" valign="top">
+                <p align="right">
+                    get_file_properties
+                </p>
+            </td>
+            <td width="111" valign="top">
+                <p align="right">
+                    FileService
+                </p>
+            </td>
+            <td width="163" valign="top">
+                <p align="right">
+                    get_file_properties
+                </p>
+            </td>
+            <td width="107" valign="top">
+                <p align="right">
+                    ShareFileClient
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td width="242" valign="top">
+                <p align="right">
+                    exists
```

Comment:
Interestingly we have `DirectoryClient.exists` but we do not have exists for share or file. Probably a gap we should close. I guess here you can just link to `exists` in the `DirectoryClient` for now?